### DISCUSSION TITLE
Local Achievement Definition Backups

### DISCUSSION THREAD

**Discussion by redwizard42** [RetroAchievements/RAIntegration#946] [Ideas]
Working through tickets in Devember, we are supposed to back up logic. Usually I do this right away in the ticket itself.  Once or twice I forgot, and then tried to re-create the original code so I could copy def...

I think it would be really nice if when we save an achievement, a local backup/repository of the prior definition were to be saved locally.  It'd then be nice if we could look back/open the older definitions.  Some concern with when/how this should be pruned, but overall this would be very helpful, even when working on our own logic in local.  Every now and then I do a code overhaul and realize my old attempt was better, and then have to re-created it.

**Comment by Jamiras** [comment]
There's a discussion about doing this on the server here: https://github.com/RetroAchievements/RAWeb/discussions/1036

**Reply by redwizard42** [reply]
Great! The server is most ideal; thought I would also like to make sure any local-only work would benefit.
